Tasks/UseDotNetV2/versionfetcher.ts

```diff
@@ -151,7 +151,18 @@ export class DotNetCoreVersionFetcher {
 
                     let versionInfoList: VersionInfo[] = [];
                     channelReleases.forEach((release) => {
-                        if (release && release[packageType] && release[packageType].version) {
+                        if (release && packageType === 'sdk' && release.sdks) {
+                            try {
+                                release.sdks.forEach((sdk) => {
+                                    let versionInfo: VersionInfo = new VersionInfo(sdk, packageType);
+                                    versionInfoList.push(versionInfo);
+                                });
+                            }
+                            catch (err) {
+                                tl.debug(tl.loc("VersionInformationNotComplete", release[packageType].version, err));
+                            }
+                        }
+                        if (release && release[packageType] && release[packageType].version && !versionInfoList.find((versionInfo) => { return versionInfo.getVersion() === release[packageType].version }) ) {
```

Comment:
Is this just to handle future scenarios where sdks might not be present?

Reply:
Otherwise if sdks is super-set, you can just do else if here

Reply:
I have kept it such as `sdks` is not documented so I am not sure if it will always remain a superset. But it looks like one.
I have raised a issue to get clarity about the property and its future.